api/src/main/java/com/rbmhtechnology/vind/api/query/facet/TermFacetOption.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2018 Redlink GmbH.
+ */
+package com.rbmhtechnology.vind.api.query.facet;
+
+/**
+ * Created on 05.11.18.
+ */
+public abstract class TermFacetOption {
```

Comment:
Why Subclassing. TermFacet Options are well defined and limited. Could be one class with 
```
hasPrefix
getPrefix
```
and later be extended to *mincount* etc. This will also solve the possible NPE mentioned above.